scio-smb/src/main/scala/com/spotify/scio/smb/SortedBucketIO.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.spotify.scio.smb
+
+import com.spotify.scio.coders.Coder
+import com.spotify.scio.io.{KeyedIO, TapOf, TapT, TestIO}
+import com.spotify.scio.util.ScioUtil
+
+final class SortedBucketIO[K, T](path: String, override val keyBy: T => K)(implicit
```

Comment:
now that I think about it, this class name is a bit confusing as it conflicts with the beam packaged SortedBucketIO https://github.com/spotify/scio/blob/main/scio-smb/src/main/java/org/apache/beam/sdk/extensions/smb/SortedBucketIO.java. I was testing out a custom SortedBucketIO.Read override, but since I had `import com.spotify.scio.smb._` in my job (in order to get the Scala API bindings for reads/writes), I had to rename the `org.apache.beam.sdk.extensions.smb.SortedBucketIO` import to be able to use it

Maybe `SmbIO`? `SortedBucketTestIO`?

Reply:
Yes I noticed that. I tried to keep consistency with java naming but It's probably to better to avoid conflict with `SmbIO`

Reply:
cool. I can make that change today!